app/components/node/connect/node-connect.html

```diff
@@ -0,0 +1,36 @@
+<div class="slideout-form content-wrapper node-connect" *ngIf="node && pool">
+    <div class="connect-content">
+        <h1>Connect to node </h1>
+        <div *ngIf="credentialSource === null">
+            <h3>Node credentials</h3>
+            <div class="credentials-source">
+                <bex-submit-btn [submit]="generateCredentials">Generate</bex-submit-btn>
+                <bex-submit-btn [submit]="specifyCredentials">Specify</bex-submit-btn>
+            </div>
+        </div>
+        <div *ngIf="credentialSource === CredentialSource.Specified && !credentials">
+            <bex-node-user-credentials-form [submit]="addOrUpdateUser" [isWindowsNode]="isWindows"></bex-node-user-credentials-form>
+        </div>
+        <div>
+            <bex-property-group *ngIf="credentials" class="normal">
+                <div class="warn-text">Those credentials are now valid for 24 hours on this node</div>
```

Comment:
I'm guessing that "These" would be more appropriate than "Those" in this context. Those is normally used in the more distant of the options. 

eg: "These are the current items, whereas those are the other ones."